pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -178,7 +180,7 @@ func (s *Service) createInstance(machine *actuators.MachineScope, bootstrapToken
 				SaKey:            string(s.scope.ClusterConfig.SAKeyPair.Key),
 				BootstrapToken:   bootstrapToken,
 				ELBAddress:       s.scope.Network().APIServerELB.DNSName,
-			})
+			}, v1beta1.JoinConfiguration{})
```

Comment:
Need to pass the user specified config int this function instead of an empty config